dev-requirements.txt

```diff
@@ -1,4 +1,3 @@
--e git+https://github.com/cloudify-cosmo/cloudify-dsl-parser@master#egg=cloudify-dsl-parser==3.2a1
--e git+https://github.com/cloudify-cosmo/cloudify-rest-client@master#egg=cloudify-rest-client==3.2a1
--e git+https://github.com/cloudify-cosmo/cloudify-plugins-common@master#egg=cloudify-plugins-common==3.2a1
-nose
```

Comment:
It was causing intellij tell me i was failing unittests because ofconflicting imports